common/postgres-init.sql

```diff
@@ -1,3 +1,4 @@
 CREATE DATABASE catalog_db;
+\c catalog_db;
```

Comment:
since this is a psql command and not sql it doesn't need a `;`
```suggestion
\c catalog_db
```